wagtail/search/management/commands/update_index.py

```diff
@@ -54,25 +54,30 @@ def group_models_by_index(backend, models):
 
 
 class Command(BaseCommand):
+    def print(self, *args, **kwargs):
```

Comment:
Sorry to be annoying but can we name this `write` or something other than `print` - it looks like a potential mistake if we use `self.print`. 

I know the original PR suggested this method name but I think we should give it a different name.